docs/installation/install-client.rst

```diff
@@ -57,6 +57,19 @@ On Ubuntu 14.04 (64-bit), the Flocker CLI can be installed from the ClusterHQ re
 
 .. begin-body-installing-client-linux
 
+Installing on RHEL 7.2
```

Comment:
These instructions have been embedded into the `installing-client-linux` section - I'm not sure if this was intentional - it probably makes sense unless there is a scenario where we would want to publish instructions for "other linux distributions" but not RHEL. 

As it stands, every version of our install instructions (Docker/Kubernetes/Standalone) all have RHEL in the installing the client, but not all of them have RHEL in the node instructions (see my previous comment)